GNUmakefile

```diff
@@ -1,19 +1,14 @@
-TEST?=$$(go list ./... |grep -v 'vendor')
+TEST?=$$(go list ./...)
```

Comment:
We don't need to filter out `vendor`. `go list ./...` doesn't print packages in `vendor/` anymore.